### PR TITLE
Adding the VR_ID and Genomic_HGVS_38 and 38 fields.  Reordering for c…

### PR DESCRIPTION
**Pull request by melissacline**
…onsistency with built_with_change_types.tsv/

### CODE REVIEW

File: pipeline/built_with_change_types.dictionary.tsv [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Field Name	Description	Example Value	Useful Resources
-Source	One or more labs, repositories, consortiums, or other entities who reported data for the variant (does not include Submitters to ClinVar)	ENIGMA,ClinVar,LOVD,ESP,1000_Genomes,ExAC,BIC,exLOVD
```
Comment (zfisch) [inline pipeline/built_with_change_types.dictionary.tsv:2]: was BIC removed here intentionally? how come?

Comment (melissacline) [reply pipeline/built_with_change_types.dictionary.tsv:2]: Yes, I removed it intentionally because it's deprecated as a data source.

Comment (zfisch) [reply pipeline/built_with_change_types.dictionary.tsv:2]: I'm unclear... we still collect it in our pipeline runs, label variants as such in the database, and provide a filter in the UI for it

Comment (melissacline) [reply pipeline/built_with_change_types.dictionary.tsv:2]: Hmm, I see what you mean.  For consistency, we should either include BIC here or exclude it everywhere.  I'll add it back in.

Comment (melissacline) [reply pipeline/built_with_change_types.dictionary.tsv:2]: Duly updated.

**Pull request merged**